pgjdbc/src/test/java/org/postgresql/test/jdbc2/BatchedInsertReWriteEnabledTest.java

```diff
@@ -153,16 +255,39 @@ public void testBatchWithReWrittenBatchStatementWithSemiColon()
       pstmt.setString(2, "c");
       pstmt.setInt(3, 6);
       pstmt.addBatch();
-      assertTrue(
-          "Expected outcome not returned by batch execution.",
-          Arrays.equals(new int[] { Statement.SUCCESS_NO_INFO,
-              Statement.SUCCESS_NO_INFO, Statement.SUCCESS_NO_INFO },
-              pstmt.executeBatch()));
+      Assert.assertEquals(
+          "Three batched statements should be rewritten to single multi-values one, thus expecting SUCCESS_NO_INFO",
```

Comment:
@vlsi I don't understand this test case. The original idea was to not perform any rewrite if there is more than 1 opening or closing parenthesis after the VALUES clause.